index.html

```diff
@@ -38,7 +38,63 @@ <h3 class="nav__logo__title">ASOCIAȚIA OPORTUNITĂȚI ȘI CARIERE</h3>
       <section class="hero"></section>
       <section class="our-mission"></section>
       <section class="contribution"></section>
-      <section class="how-contribute"></section>
+      <section class="how-contribute">
+        <div class="how-contribute__title">
+          <h2>Cum poți contribui la echipă?</h2>
+          <img src="/assets/underline.svg" alt="" />
+        </div>
+        <div class="how-contribute__description">
+          <p>
+            Cu determinare și cu implicarea unor voluntari pasionați, am reușit
+            să aducem laolaltă locuri de muncă de la peste 700 de companii,
+            numărul acestora crescând.
+          </p>
+          <p>
+            Cu toate acestea, întrucât în fiecare zi apar locuri de muncă noi,
+            avem mereu nevoie de oameni care să ne ajute să menținem și să
+            dezvoltăm această platformă.
+          </p>
+          <p>
+            Găsește rolul potrivit pentru tine și fă o diferență în comunitate!
+          </p>
+        </div>
+        <div class="how-contribute__jobs">
+          <img src="./assets/arrow-jobs.svg" alt="arrow-left" id="arrow-left" />
+          <img
+            src="./assets/arrow-jobs.svg"
+            alt="arrow-right"
+            id="arrow-right"
+          />
+          <div class="how-contribute__job">
+            <img src="./assets/fe-job.png" alt="" />
+            <div class="how-contribute__job__text">
+              <h3>Frontend Developer</h3>
+              <a href="">Detalii ›</a>
+            </div>
+          </div>
+          <div class="how-contribute__job">
+            <img src="./assets/be-job.png" alt="" />
+            <div class="how-contribute__job__text">
+              <h3>Frontend Developer</h3>
+              <a href="">Detalii ›</a>
+            </div>
+          </div>
+          <div class="how-contribute__job">
+            <img src="./assets/ba-job.png" alt="" />
```

Comment:
si la aceste alt-uri de la img tag-uri la fel putem pune ceva in acel alt, la caz ca imaginea nu exsita sa afisam un mesaj ceva in UI.